uk.co.compucorp.civicrm.tasksassignments/CRM/Activity/Form/ActivityView.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
since this is an overridden files too .. the same notes I left in the other PR apply to it .